src/test/java/org/broadinstitute/hellbender/tools/walkers/bqsr/DF_BaseRecalibratorIntegrationTest.java

```diff
@@ -33,7 +33,7 @@ private BQSRTest(String reference, String bam, String knownSites, String args, S
         public String getCommandLine() {
             return  " -R " + reference +
                     " -I " + bam +
-                    " --secretsFile ../client-secrets.json" +
+                    " --secret ../client-secrets.json" +
```

Comment:
Could you change this so that it looks in the current working directory for the `client-secrets.json`, rather than the parent directory? Our current practice is to keep our `client-secrets.json` in our hellbender root.